homeassistant/components/roborock/image.py

```diff
@@ -130,25 +146,42 @@ async def create_coordinator_maps(
     maps_info = sorted(
         coord.maps.items(), key=lambda data: data[0] == cur_map, reverse=True
     )
-    for map_flag, map_name in maps_info:
+    maps = await asyncio.gather(
+        *(roborock_storage.async_load_map(map_name) for map_name in coord.maps.values())
```

Comment:
To avoid swamping the executor with jobs, can you add an `async_load_maps` function, that only executes a single executor job.

Reply:
Actually, this only happens during startup right? Shouldn't this just be an `async_initalize` ?

Reply:
It does only happen on startup. I'm not sure I know what async_initalize is though. I'm looking around at other references, but i'm not 100% following how it applies here/ how to do it.